app/src/main/java/com/infomaniak/mail/utils/FetchMessagesManager.kt

```diff
@@ -135,6 +136,15 @@ class FetchMessagesManager @Inject constructor(
             return
         }
 
+        // Dismiss Notifications for Messages that have been read on another device
```

Comment:
If we could also make this feature trigger when the user unlock is phone, it would be wonderful !

Reply:
It will be done in another PR